src/syntax-interpreter.js

```diff
@@ -130,13 +130,16 @@ export class SyntaxInterpreter {
     let ii;
     let inString = false;
     let inEscape = false;
+    let foundNameValuePair = false;
+    let type;
 
     for (i = 0, ii = attrValue.length; i < ii; ++i) {
       current = attrValue[i];
 
       if (current === ';' && !inString) {
         info = language.inspectAttribute(resources, '?', name, target.trim());
         language.createAttributeInstruction(resources, element, info, instruction, context);
+        foundNameValuePair = true;
```

Comment:
I think we can move the `foundNameValuePair = true;` line into the `} else if (current === ':' && name === null) {` block. Then the `if (!foundNameValuePair && (name === null)) {` statement can be simplified to `if (!foundNameValuePair) {`.

The way it's currently written it seems like `foundNameValuePair` can be false when the attribute value is `foo: bar` because the options binding expression has no semicolon.